README.md

```diff
@@ -290,6 +298,71 @@ order to use EL function mapping, the folder `o2a_libs` should
+might be removed in the future. Here is a list of common known limitations that we are aware of for now:
+
+Many of those limitations are not blockers - the workflows will still be converted to Python DAGs
+and it should be possible to manually (or automatically) post-process the DAGs to add custom
+functionality. So even with those limitations in place you can still save a ton of work when
+converting many Oozie workflows.
+
+In the following, "Examples" section more specific per-action limitations are listed as well.
+
+## File/Archive functionality
+
+At the time of this writing we were not able to determine if file/archive
+functionality works as intended. While we map appropriate file/archive methods it seems that Oozie
+treats file/archive somewhat erraticaly. This is not a blocker to run most of the operations, however
+some particular complex workflows might be problematic. Further testing with real, production Oozie
+workflows is needed to verify our implementation.
+
+* [File/Archive in Pig doesn't work](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/243)
+
+## Not all global configuration methods are supported
+
+Oozie implements a number of ways how configuration parameters are passed to actions. Out of the existing
+configuration options the following ones are not supported (but can be easily added as needed):
+
+* [The config-default.xml file](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/137)
+* [Parameters section of workflow.xml](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/138)
+* [Handle Global configuration properties](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/134)
+
+## Support for uber.jar feature
+
+The uber.jar feature is not supported.
+
+* [Support uber.jar feature](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/140)
+
+## Support for .so and .jar lib files
+
+Oozie adds .so and .jar files from the lib folder to Local Cache for all the jobs run to
+LD_LIBRARY_PATH/CLASSPATH. Currently only Java Mapper supports it.
+
+* [Support for Oozie lib .so files](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/302)
+* [Support for Oozie lib .jar files](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/301)
+
+## Custom messages missing for Kill Node
+
+The Kill Node might have custom log message specified. This is not implemented:
+[Add handling of custom Kill Node message](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/97)
+
+## Capturing output is not supported
+
+In several actions you can capture output from tasks. This is not yet implemented.
+
+* [Add support for capture-ouput](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/155)
+
+## Subworkflow DAGs must be placed in examples
+
+Currently all subworkflow DAGs must be in examples folder
+
+* [Subworkflow conversion expects to be run in examples](https://github.com/GoogleCloudPlatform/oozie-to-airflow/issues/213)
+
```

Comment:
Should we add info about EL functions? I can prepare it 🙂

Reply:
I think it is already done in your PR, isn't it :) ?